data/112/528/044/5/1125280445.geojson

```diff
@@ -175,6 +189,8 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
+        "eg:gisco_id":"LU_0202",
+        "eurostat:nuts_2021_id":"0202",
```

Comment:
Ah, same thing about type values here - should these be ints?

Reply:
Like US Census GEOID, the leading `0` is significant and used to pad shorter ints to a fixed character width so you can glance at the length and know what type of feature it is. A clever solution that's now annoying, but it is what it is.